samples/matter/common/src/bridge/ble_bridged_device.h

```diff
@@ -76,8 +77,35 @@ class BLEBridgedDeviceProvider : public BridgedDeviceDataProvider {
 
 	bt_addr_le_t GetBtAddress() { return mDevice.mAddr; }
 
+	/**
+	 * @brief Get a number of failed recovery attempts for this provider.
+	 */
+	uint16_t GetFailedRecoveryAttempts() { return mFailedRecoveryAttempts; }
+
+	/**
+	 * @brief Inform provider that recovery attempt for it failed.
+	 *
+	 * This method increments the number of failed recovery attempts.
+	 *
+	 */
+	void NotifyFailedRecovery()
```

Comment:
We didn't notify anyone within this method, right?  What about renaming it to `IncreaseRecoveryAttempts`

Reply:
I mean, notify the provider that recovery attempt failed/succeeded, so internally it may take some actions due to that. I actually followed the naming scheme used by you in the `Recovery`. We have there `NotifyProviderToRecover` method that is used to notify `Recover` module that new provider was lost.